django/core/checks/registry.py

```diff
@@ -17,6 +17,7 @@ class Tags(object):
     models = 'models'
     security = 'security'
     signals = 'signals'
+    staticfiles = 'staticfiles'
```

Comment:
@adamchainz  Thank you! The tag is replaced as the string 'staticfiles':

https://github.com/django/django/pull/7781/files#diff-b0bf672a5585eec6d67d88f4d378b9afR13